src/content/development/contribution-guide/_index.en.md

```diff
@@ -66,7 +66,17 @@ To be expanded
 
 #### Development Milestones
 
-To be expanded
+The milestone sprints are focused on development, encapsulating work on various [tasks](#tasks) and [bugs](#bugs).
+Project members will work on any planned tasks and bugs, and will also work on unplanned bugs should they arise.
+Any [unplanned work](#unplanned-work) will have to follow the defined guidelines.
+
+Each sprint will end with a milestone release which allows the project community to test newly added features and fixed bugs.
+In total, three milestone sprints are planned:
+
+* Two milestones ending with releases of milestone `m1` and milestone `m2`.
+* The last sprint will end with the release of the [release candidate](#release-candidates) `rc0`.
+
+As detailed in the [sprints](#sprints) section, each milestone releases will be followed by a test day and the sprint meetings.
```

Comment:
```suggestion
As detailed in the [sprints](#sprints) section, each milestone release will be followed by a test day and the sprint meetings.
```

Do we also want to mention documentation days? Or do we want to avoid “enshrining” them, and reinforce documentation as part of the definition of done instead? (I think there’s value in separate documentation tests since they give a more comprehensive review of the documentation as a whole.)

Reply:
Docs day will be happening after every release just like test day.

Reply:
whatever makes sense :)

we didnt actually discuss doc days so theres no consensus. to me doc days make more sense to do after the rc0, if we do them, since docs should be updated as part of "definition of done" anyhow and doc day  is more of a documentation review and alignment IMHO

also we didnt plan a "doc day" section so if we want to have them, maybe add this info later as it also requires changes in other sections

Reply:
Documentation should be reinforce as part of DoD.
We can discuss documentation for each epic/story at the Sprint review meeting at end of each sprint.